initfiles/componentfiles/configxml/ldapserver.xsd

```diff
@@ -169,6 +169,15 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+
+            <xs:attribute name="sharedCache" type="xs:boolean" use="optional" default="true">
```

Comment:
I made it configurable because I cannot load test like it would be used in a production environment. If it turns out to be slower, which I seriously doubt, they could revert back to the legacy behavior.  Once we are satisfied with it, we can easily remove the configuration option